womenwhogo.org/scholarships.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Women Who Go scholarship application">
+    <meta name="keywords" content="Women Who Go, Go, Golang, Women in Technology, WWG, Go, Programming, Events, scholarship">
+
+    <meta name="google-site-verification" content="D_yba5vBXKgsE50Rcl00dhHgrHZttf2zB0-TSAYW8hY" />
+
+    <link rel="icon" href="favicon.ico">
+
+    <title>Scholarhips - Women Who Go</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
+
+    <link href="./assets/css/index.css" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="//oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <script src="//oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <div class="container header">
+      <div class="row"><p class="text-center col-xs-6 col-xs-offset-3">
+        <a href="/">
+          <img class="main-img" src="assets/images/wwglogo.png">
+        </a>
+      </p></div>
+      <h1 class="text-center">GopherCon Scholarhip</h1>
+    </div>
+    <hr>
+    <br>
+    <br>
+    <div class="container links">
+      <h2>Eligibility</h2>
+        <p>You must meet at least one of the following requirements:</p>
+        <ul>
+          <li>Identify as a woman or non-binary</li>
+          <li>Identify as a person of color</li>
+          <li>Identify as LGBTQ+</li>
```

Comment:
thinking of making it `LGBTQIA+` but I could be biased on this since I am part of the `A`

Reply:
Can we fix that on the form too? I was copying what we used there. I'll fix on the page for now.